CONTRIBUTING.md

```diff
@@ -1,86 +1,187 @@
 
-[examples]: https://github.com/pact-foundation/pact-python/tree/master/examples
+### Join our Slack
 
-## Commit messages
+We have a [Slack](https://slack.pact.io) to discuss all things about Pact and its development. Feel free to ask questions about Pact Python specifically in the [`#pact-python`](https://pact-foundation.slack.com/archives/C9VECUP6E) channel, or broader questions about the Pact ecosystem over in the [`#general`](https://pact-foundation.slack.com/archives/C5F4KFKR8) channel.
 
-pact-python is adopting the [Conventional Commits](https://www.conventionalcommits.org)
-convention. Please ensure you follow the guidelines, we don't want to be that person,
-but the commit messages are very important to the automation of our release process.
+### Triaging Issues and Pull Requests
 
-Take a look at the git history (`git log`) to get the gist of it.
+One great way you can contribute to the project without writing any code is to help triage issues and pull requests as they come in.
 
-If you'd like to get some CLI assistance there is a node npm package. Example usage is:
+-   Ask for more information if you believe the issue does not provide all the details required to solve it.
+-   Suggest [labels](https://github.com/pact-foundation/pact-python/labels) that can help categorize issues.
+-   Flag issues that are stale or that should be closed.
+-   Ask for test plans and review code.
 
-```shell
-npm install -g commitizen
-npm install -g cz-conventional-changelog
-echo '{ "path": "cz-conventional-changelog" }' > ~/.czrc
-```
+## Our Development Process
 
-When you commit with Commitizen, you'll be prompted to fill out any required
-commit fields at commit time. Simply use `git cz` or just `cz` instead of
-`git commit` when committing. You can also use `git-cz`, which is an alias
-for `cz`.
+Pact Python uses [GitHub](https://github.com/pact-foundation/pact-python) as its source of truth. The team will be working directly there. All changes will be public from the beginning.
 
-See https://www.npmjs.com/package/commitizen for more info.
+All pull requests will be checked by the continuous integration system, GitHub actions. There are unit tests, end-to-end tests, performance tests, style tests, and much more.
 
-There is a pypi package that does similar [commitizen](https://pypi.org/project/commitizen/).
+### Branch Organization
 
-## Running the tests
+Pact Python has one primary branch `master` and we use feature branches to deliver new features with pull requests. We use the following naming convention for branches:
 
-You can run the tests locally with `make test`, this will run the tests with `tox`
+-   `feature/<name>` or `feat/<name>` for new features
+-   `fix/<name>` for bug fixes
+-   `chore/<name>` for chores
+-   `docs/<name>` for documentation changes
```

Comment:
are there more styles? commitlint provides about 8 I think from the precommit-lint error message. `refactor` etc

good shout to call out here

Reply:
I personally would put `refactor` in the same type as `style` 😆 

I've reworded the paragraph to mention that it should be linked to the conventional commit type.

Reply:
:)  ahh see style isn't in the list so that is what made me think.

Thanks for the reword!